src/main/scala/Column.scala

```diff
@@ -1057,12 +1194,25 @@ case class Column private (
               s"'$v' - ${errorMessageContent.content}",
               datatype.toPrettyString
             )
+            val invalidNode = JsonNodeFactory.instance.objectNode()
+            invalidNode.set("invalid", new TextNode(v))
+            valuesArrayToReturn = valuesArrayToReturn :+ invalidNode
+          }
+          case Right(s) => {
```

Comment:
Why do we want to return JsonNodes in here?

Reply:
This is something which I was unsure while I wrote it. The csvlint was returning an invalid key value where the column values does not pass the validation and thus I did this.

Reply:
I think since we're returning parsed values here, we should simply return a string saying `s"invalid - $v"`. That'll give us the same amount of information to present to the user.

I suppose really what they want to do here is provide a warning instead of an error? Perhaps we can change the function so we return both errors and warnings using an object similar to the `ErrorsAndWarnings` one we have now?

Reply:
> so I think setting a scala string value should be sufficient here

an empty string `""` ?

Reply:
`s"invalid - $v"` I think would be most consistent and probably useful.

Reply:
👍